features/openhab-addons-external/src/main/resources/conf/dsmr.cfg

```diff
@@ -0,0 +1,11 @@
+############################## DSMR Binding ################################
+#
+# Port of the DSMR port (mandatory, e.g. /dev/ttyUSB0)
+#port=/dev/ttyUSB0
+# Configuration of additional meters (channel 0 is used for the main electricity meter)
+#gas.channel=1
+#water.channel=2
+#heating.channel=3
+#cooling.channel=4
+#generic.channel=5
+#slaveelectricity.channel=6
```

Comment:
See earlier comment about which source of config information -- ESH-INF/bindinng.xml vs. dsmr.cfg -- is used.